package.json

```diff
@@ -2,7 +2,13 @@
   "name": "skynet-js",
   "version": "4.0.4-beta",
   "description": "Sia Skynet Javascript Client",
-  "main": "dist/index.js",
+  "type": "module",
+  "main": "dist/cjs/entry.js",
+  "module": "dist/mjs/index.js",
+  "exports": {
```

Comment:
Can you explain this please? Does this PR add `js/` and `mjs/` directories?

Reply:
So "module" format, or ESM will be in `./dist/mjs/` and commonJS in `./dist/cjs/` -- the package.json should point projects in node or browserjs to the correct dir for their import types. I think this also means when we modify browser-specific code to node, we can just target the commonjs build.

I'm following the lead of https://www.sensedeep.com/blog/posts/2021/how-to-create-single-source-npm-module.html and a few other assorted resources.

Reply:
There is no longer a `./dist/js` dir, but you will see right away all the `require` statements in the cjs files.

Reply:
Misunderstood you initial comment, but this should be resolved by matching the article in commit 11537c0